Classes/Domain/Service/UserLocaleService.php

```diff
@@ -37,6 +37,13 @@ class UserLocaleService
      */
     protected $rememberedContentLocale;
 
+    /**
+     * Remebered content locale for locale switching
+     *
+     * @var Locale
+     */
+    protected $rememberedUserLocale;
```

Comment:
Are those changes related?

Reply:
Not a 100%, just some optimisation as the locale is switched twice for every node triggered by the augmentation